exampleCourse/questions/element/dragDropBlocks/question.html

```diff
@@ -0,0 +1,42 @@
+
+<p>
+Drag all the prime numbers below to the yellow box. Note that the ordering of the answer does not matter, any permutation of prime numbers are accepted. Remember 1 is not a prime number.
+</p>
+
+<pl-order-blocks answers-name="prime-numbers-unordered" shuffle-options="true" permutation-mode="any">
```

Comment:
We could definitely obscure the feedback at the bottom, but I think it might be harder to hide the percentage in the `partially correct: x%`, since PL handles that? That element is not part of the submission panel

Reply:
what about removing 

```
        {{^perfect_score}}
                <span class="badge {{colour}}"><i class="fa fa-times" aria-hidden="true"></i>
        {{score}}%</span> {{feedback}}
        {{/perfect_score}}
```

in the mustache file. I think that partials is not needed there. 

@mwest1066, @cheeren what is your opinion about this?